manifest/pack_pull_git.go

```diff
@@ -188,7 +188,30 @@ func pullGit(ctx context.Context, manifest *Manifest, opts ...pack.PullOption) e
 		WithField("branch", popts.Version()).
 		Infof("git clone")
 
-	if _, err = git.PlainCloneContext(ctx, local, false, copts); err != nil {
+	_, err = git.PlainCloneContext(ctx, local, false, copts)
+	switch err {
+	case git.ErrRepositoryAlreadyExists:
```

Comment:
This won't work anymore if the returned error is ever wrapped upstream.
Since Go 1.13, the recommended way is to use `errors.Is` to assert a certain error type (or `errors.As` if the underlying dynamic type needs to be accessed).